adapters/rubicon/rubicon.go

```diff
@@ -750,8 +749,15 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ada
 			continue
 		}
 
-		resolvedBidFloor, resolvedBidFloorCur := resolveBidFloorAttributes(thisImp.BidFloor, thisImp.BidFloorCur)
-		thisImp.BidFloorCur = resolvedBidFloorCur
+		resolvedBidFloor, err := resolveBidFloor(thisImp.BidFloor, thisImp.BidFloorCur, reqInfo)
+		if err != nil {
+			errs = append(errs, &errortypes.BadInput{
+				Message: fmt.Sprintf("Unable to convert provided bid floor currency from %s to USD",
+					thisImp.BidFloorCur),
+			})
+			continue
+		}
+		thisImp.BidFloorCur = "USD"
```

Comment:
If a given request comes with:
```
        "bidfloor": 0.00,
        "bidfloorcur": "EUR",
```
Line 760 above will change the "EUR" value to "USD" anyways. Is that ok?